website/docs/dbt-cli/installation.md

```diff
@@ -1,32 +1,133 @@
 ---
-title: "Installation"
+title: "Install dbt"
 id: "installation"
 ---
 
-We recommend you use one of our OS-specific guides for help with installing dbt. If your OS is not listed, or you prefer to use `pip`, please see the instructions below.
+We recommend you use install dbt using one of three tried and tested methods:
 
-## OS-specific guides
-- [macOS](macos)
-- [Windows](windows) 
-- [Ubuntu / Debian](ubuntu-debian)
-- [CentOS](centos)
+- [brew](#homebrew) (recommended for MacOS)
+- [pip](#pip)
+- [from source](#install-from-source)
+
+## Homebrew
+
```

Comment:
Can we add a tip here, like "this is the recommended way of installing for MacOS users"?

Reply:
Oh, I see it above :)